### ISSUE TITLE
small bug: change of fps value from SpriteAnimation only applied when pressing arrow buttons

### ISSUE THREAD

**Issue by kutoman** [UnderwaterApps/overlap2d#156]
the return key has no effect

**Comment by kutoman** [comment]
I will fix that right now

**Comment by azakhary** [comment]
I think this issue is for all cases we use VisUI NumberSelector, or rather on the Listener we put on it. If you fix it, it will basically get fixed for all.

**Comment by kutoman** [comment]
I checked the **UISpriteAnimationItemProperties**. The passed listener is not a common listener:
```
fpsSelector.addChangeListener(number -> facade.sendNotification(getUpdateEventName()));
```
Shall I create one common listener?

**Comment by azakhary** [comment]
Yeah I can see we are using Change listener. maybe if we update to latest version, it can fix it. lemme check.

**Comment by azakhary** [comment]
Wait. Enter works on label size NumebrSelector

**Comment by azakhary** [comment]
And on animations as well

**Comment by azakhary** [comment]
Okay it works on labels, does not work on FPS, so it's us.

**Comment by azakhary** [comment]
fixed in #160
